portfolio/src/main/webapp/index.html

```diff
@@ -11,7 +11,35 @@
       <h1>Stefano's Portfolio</h1>
       <img src="/images/selfie.jpg" alt="Stefano's selfie" width="250">
       <p>I am Stefano and I am from Italy.</p>
-      <p>I am pursuing a bachelor's degree in Computer Science and Engineering at the Politecnico di Torino and I am in the second year.</p>
+      <div id="education">
```

Comment:
Note that using ids in HTML for styling (which you'll probably do later) can have unwanted side-effects and come with some challenges. Especially since you can define only one id vs multiple classes for an element and because it has to be unique over the whole page. 

Some interesting resources discussing this (with different suggestions):
- https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4
- https://stackoverflow.com/questions/1878810/is-there-any-pros-and-cons-if-i-use-always-css-class-instead-css-id-for-everythi